resources/zib/zib-AdministrationAgreement.xml

```diff
@@ -215,6 +215,7 @@
       <path value="MedicationDispense.performer.actor" />
       <short value="Supplier" />
       <definition value="The supplier (pharmacist) that entered the administration agreement." />
+      <comment value="The 'Supplier' concept of zib AdministrationAgreement is mapped to both `performer.actor` and `location`. Because individuals who enter the administration agreement are not recorded, the organizational information of the supplier (pharmacist) can be referenced here." />

```

Comment:
Ik ben bang dat ik het tweede deel van deze tekst niet snap. Het zib concept is toch sowieso al een HealthcareProvider en geen persoon?

Reply:
Dit is eigenlijk zelfs een verantwoording waarom in de zib naar Zorgaanbieder in plaats van Zorgverlener verwezen wordt. In eerdere versies van de zibs is Supplier een verwijzing naar Zorgverlener (dat sijpelt af en toe in de tekst nog door - 'apotheker' ipv 'apotheek', al is 'apotheker' ook een prima woord voor de Organization denk ik). Maar omdat apothekers (hier doe ik het zelf ook) geen personen registreren is zelfs de zib aangepast. Dit kan scherper inderdaad.